neuralop/layers/discrete_continuous_convolution.py

```diff
@@ -416,7 +447,10 @@ class DiscreteContinuousConvTranspose2d(DiscreteContinuousConv):
         Can also pass a string to generate a regular (tensor) grid.
         For exact options see ``torch_harmonics.quadrature``.
     kernel_shape: Union[int, List[int]]
-        kernel shape. Expects either a single integer for isotropic kernels or two integers for anisotropic kernels
+        kernel shape. Expects either a single integer for isotropic kernels
+        or two integers for anisotropic kernels
```

Comment:
The formatting is weird. Also made me realize this description isn't super helpful. Can we explain the two terms and add more detail (or better, an example) for each cases?

Reply:
Updated